__tests__/integration.test.js

```diff
@@ -251,4 +251,33 @@ describe('patchArticle()', () => {
         expect(body.msg).toBe('Bad Request');
       });
   });
-});
\ No newline at end of file
+});
+
+describe('deleteComment()', () => {
+  test("204: delete comment by it's comment_id", () => {
+    return request(app)
+      .delete('/api/comments/1')
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toMatchObject({});
+      });
+  });
+
+  test('POST:400 sends a 400 + error message when invalid id is provided', () => {
+    return request(app)
+      .delete('/api/comments/apple')
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Bad Request');
+      });
+  });
+
+  test('POST:404 sends a 404 + error message when missing id', () => {
```

Comment:
This needs changing to DELETE and it shouldn't be concerned about a missing id it should however be concerned about an ad that is not found